spec/component/models/publication_spec.rb

```diff
@@ -460,6 +460,51 @@
     end
   end
 
+  describe 'other scopes' do
+    let!(:pub1) { create(:publication,
+                         title: 'pub1')
+    }
+    let!(:pub2) { create(:publication,
+                         title: 'pub2',
+                         doi_verified: false)
+    }
+    let!(:pub3) { create(:publication,
+                         title: 'pub3',
+                         doi_verified: true)
+    }
+    let!(:pub4) { create(:publication,
+                         title: 'pub4',
+                         doi_verified: nil)
+    }
+    let!(:pub5) { create(:publication,
+                         title: 'pub4',
+                         doi_verified: nil)
+    }
+    let!(:activity_insight_oa_file1) { create(:activity_insight_oa_file, publication: pub2) }
+    let!(:activity_insight_oa_file2) { create(:activity_insight_oa_file, publication: pub3) }
+    let!(:activity_insight_oa_file3) { create(:activity_insight_oa_file, publication: pub4) }
+
+    let!(:open_access_location) { create(:open_access_location, publication: pub5) }
+
+    describe '.not_open_access' do
+      it 'returns publications that do not have open access information' do
+        expect(described_class.not_open_access.map(&:title)).to match_array [pub1, pub2, pub3, pub4].map(&:title)
+      end
```

Comment:
It's a little hard for me to tell what's going on here given that there's a publication with a duplicate title and we're comparing result sets by title. Curious what your reasoning is for doing it that way..

Reply:
The intention was to exclude pub5 since it has an open access location, but the typo is making this not quite make sense.  I don't really have a good reason for only matching the titles other than that's how the other scopes are tested above.

Reply:
Ok, cool - if it'll work fine, then I'd match on the objects/records themselves instead of their titles. I'd definitely find that to be less confusing since that's really what we want to assert here.